### PR TITLE
redirect fixed

### PR DESCRIPTION
**Pull request by otabeknos1rov**

### CODE REVIEW

**Review by IslomJonMirzakbarov — CHANGES_REQUESTED** [review]
react-query/devtools removed? I think we may need it when we start work with datas, it will be much help for development

File: apps/web/src/styles/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@mixin fontTitleXXL {
+  font-weight: 600;
+  font-size: 32px;
```
Comment (JM066) [inline apps/web/src/styles/_fonts.scss:3]: Great! You categorized this. Would it be possible to use rem instead for the font size here?

Comment (otabeknos1rov) [reply apps/web/src/styles/_fonts.scss:3]: Yes it is , for clearness it is better to keep similar with style guideline from figma, isn't it?

**Comment by otabeknos1rov** [comment]
> react-query/devtools removed? I think we may need it when we start work with datas, it will be much help for development

Oh , I see , yeah we can leave devtools

**Review by JM066 — APPROVED** [review]

**Review by dbjowett — APPROVED** [review]

**Pull request merged**